packages/app/src/specs/spec-utils.ts

```diff
@@ -2,10 +2,13 @@ import fuzzySort from 'fuzzysort'
 import type { FoundSpec } from '@packages/types'
 import { ComputedRef, Ref, ref, watch } from 'vue'
 import _ from 'lodash'
+import type { UseCollapsibleTreeNode } from '@packages/frontend-shared/src/composables/useCollapsibleTree'
+import { getRunnerConfigFromWindow } from '../runner'
 
-import type { UseCollapsibleTreeNode } from '../composables/useCollapsibleTree'
-
-const PATH_SEP = /[\/\\]/
+// @ts-ignore
+const platform = window.Cypress?.platform ?? getRunnerConfigFromWindow().platform
```

Comment:
```suggestion
const platform = getRunnerConfigFromWindow().platform
```

I think - `window.Cypress` won't exist here, only on the runner page, but `__CYPRESS_CONFIG__` is defined from the very moment the app is opened (see here  https://github.com/cypress-io/cypress/blob/10.0-release/packages/data-context/src/sources/HtmlDataSource.ts#L119).